dist/raven.js

```diff
@@ -457,9 +460,10 @@ Raven.prototype = {
 
         if (this._globalOptions.stacktrace || (options && options.stacktrace)) {
             var ex;
-            // create a stack trace from this point; just trim
-            // off extra frames so they don't include this function call (or
-            // earlier Raven.js library fn calls)
+            // Generate a "synthetic" stack trace from this point.
+            // NOTE: If you are a Sentry user, and you are seeing this stack frame, it is NOT indicative
+            //       of a bug with Raven.js. Sentry generates synthetic traces either by configuration,
+            //       or if it catches a thrown object without a "stack" property.
```

Comment:
we might want to make this warning more explicit directly above the line that they'll see - worried that this will be pushed out of or to the edges of context frames